app/components/Modals/GeneratedTransactionModal/GeneratedTransactionModal.jsx

```diff
@@ -119,7 +122,10 @@ export default class GeneratedTransactionModal extends React.Component<
           <div className={baseStyles.section}>
             <div className={baseStyles.sectionTitle}>TRANSACTION OUPUT</div>{' '}
             <div className={styles.transactionOutput}>
-              <pre>{JSON.stringify(this.props.tx, null, 2)}</pre>
+              <JSONPretty
```

Comment:
Was having some issues with just using the `<pre>` tag here so added this ultra weight dependency. This library also adds the ability to format JSON with CSS which could be a nice future enhancement